db/seeds.rb

```diff
@@ -0,0 +1,7 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
+#   Character.create(name: "Luke", movie: movies.first)
```

Comment:
- [ ] Here is the project requirement 👇🏻Kindly Make you create 5 different greetings, for the table for storing your messages, Actually you can use the seed to create five different greetings. ✍🏻 

![image](https://user-images.githubusercontent.com/79058364/216478580-31fba987-8dbc-4d98-a4d2-c3194ab2609e.png)